docs/lib/analytics/fragments/ios/getting-started.md

```diff
@@ -1,59 +1,65 @@
-When your backend is successfully updated there will be two newly created files: `amplifyconfiguration.json` and `awsconfiguration.json` in your project folder. The `amplifyconfiguration.json` is used for Amplify Library, while the `awsconfiguration.json` is used for the underlying AWS iOS SDK.
+## Application Setup
 
-## Install Amplify Libraries
+When your backend is successfully updated there will be two newly created files: `amplifyconfiguration.json` and `awsconfiguration.json` in your project folder. The former is used for Amplify Library, while the latter is used for the underlying AWS iOS SDK.
+
+### Cocoapods
 
 > iOS project targeting at least iOS 11.0
 
 If this is a new project, run `pod init` to create a `Podfile` for use with CocoaPods to manage your dependencies. 
 
-Add the following to your `Podfile`
+Add the following to your `Podfile`:
 
 ```ruby
-    target :'YOUR-APP-NAME' do
-        use_frameworks!
-        pod 'AmplifyPlugins/AWSPinpointAnalyticsPlugin'
-        pod 'AWSMobileClient'
-    end
+target :'YOUR-APP-NAME' do
+  use_frameworks!
+  pod 'Amplify'
```

Comment:
is Amplify really needed?

Reply:
hmm true. I almost feel like customers should pull Amplify themselves though. Relying on plugins to pull the Amplify core works well if all plugins use the same version (like all AWS Plugins). But in the use case we foresee of external plugins, what happens if a 3rd party plugin depends on a different version of Amplify?

I think this is a separate discussion we need to have though. For now I'll remove to keep it consistent with the rest of the docs.